### PR TITLE
Added formatResponse option to the ghost content API

### PR DESCRIPTION
**Pull request by nextwebb**
Added formatResponse option to the ghost content API, 
A fix for "Add an option to skip response formatting in Content/Admin API SDKs #73"
https://github.com/TryGhost/Ghost-SDK/issues/73

The contentAPI will now have the "format response" boolean property, as an option
like this:
const api = new GhostContentAPI({
  url: 'YOUR URL',
  key: 'YOUR KEY',
  version: "VERSION",
  formatResponse: true/false
})

### CODE REVIEW

**Comment by kevinansfield** [comment]
@nextwebb it seems that your PR contains a lot of file permission changes and also needs conflicts resolving. Could you look at both of those issues please?

File: packages/content-api/lib/index.js [hunk 1]
```diff
@@ -90,12 +94,27 @@ export default function GhostContentAPI({url, host, ghostPath = 'ghost', version
...
-                return res.data[resourceType][0];
+                if (formatResponse === true){
+                    return res.data[resourceType][0];
+                } else {
+                    return res.data;
+                }
+                
+            }
+            if(formatResponse === true ){
+                return Object.assign(res.dat, {meta: res.data.meta});
```
Comment (phawazzzy) [inline packages/content-api/lib/index.js:113]: this should be `return Object.assign(res.data, {meta: res.data.meta});` and not `return Object.assign(res.dat, {meta: res.data.meta});`

File: packages/content-api/lib/index.js [hunk 2]
```diff
@@ -13,7 +13,11 @@ export default function GhostContentAPI({url, host, ghostPath = 'ghost', version
     }
 
     if (this instanceof GhostContentAPI) {
-        return GhostContentAPI({url, version, key});
+        return GhostContentAPI({url, version, key, formatResponse});
+    }
+
+    if (!formatResponse) {
```
Comment (phawazzzy) [inline packages/content-api/lib/index.js:19]: I feel the formatResponse option should not throw an error if it isnt declared or assigned a value...
It should default to true just as mentioned here by the first person who commented on the issue [here](https://github.com/TryGhost/Ghost-SDK/issues/73#issuecomment-615893205)

Comment (nextwebb) [reply packages/content-api/lib/index.js:19]: I'll look into it

**Pull request closed without merging**